andela/pippin-ah-backend
# wFt/163143034/route error handler

#### What does this PR do?

-  Creates a route error-handler

#### Description of Task to be completed?

- [x] The routes error handler is a middleware and exists in the middleware directory.
- [x] Write test for the route

#### How should this be manually tested?
- start server with <code>npm start</code> and send request to a non-existent route e.g. <code>/nothing</code>.
- you should receive a 404 error with message: 'route not found'
#### Any background context you want to provide?
- none
#### What are the relevant pivotal tracker stories?
[#163143034](https://www.pivotaltracker.com/n/projects/2229853/stories/163143034)
